src/tutorials/ping.rs

```diff
@@ -136,32 +136,37 @@
 //! [plaintext](crate::plaintext). It only cares about _what_ messages are sent
 //! on the network.
 //!
+//!
```

Comment:
Why is this newline needed?

Reply:
Oops, that was by accident. Will remove it.